code/modules/client/preferences.dm

```diff
@@ -1105,6 +1147,8 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 								continue
 							if( !(species in S.species_allowed))
 								continue
+							if(species == "Machine")

```

Comment:
why are IPC's defined in species_allowed then snowflaked out of using it, why the fuck